cmake/googletest_download.txt.in

```diff
@@ -21,9 +21,8 @@ ExternalProject_Add(googletest
   SOURCE_DIR        "${GOOGLETEST_BUILD_ROOT}/googletest-src"
   BINARY_DIR        "${GOOGLETEST_BUILD_ROOT}/googletest-build"
   CMAKE_ARGS
-                    "-DCMAKE_MAKE_PROGRAM=${CMAKE_MAKE_PROGRAM}"
+                    "${MAYAUSD_EXTERNAL_PROJECT_GENERAL_SETTINGS}"
```

Comment:
Just like Maya does, use a set of general settings for all external projects (MayaUsd currently only has one for googletest).